packages/codegen-ui-react/lib/__tests__/__snapshots__/studio-ui-codegen-react-forms.test.ts.snap

```diff
@@ -634,6 +634,7 @@ import { DataStore } from \\"aws-amplify\\";
 export default function myPostForm(props) {
   const {
     id,
+    currentModel,
```

Comment:
Open to suggestions for prop variable name

Reply:
we could use the name of the model as the prop name

Reply:
Added change